### DISCUSSION TITLE
I got the issue when I run.

### DISCUSSION THREAD

**Discussion by hendrywang** [microsoft/graphrag#1069] [Q&A] [answered]
[81 rows x 5 columns]
🚀 create_base_extracted_entities
                                        entity_graph
0  <graphml xmlns="http://graphml.graphdrawing.or...
🚀 create_summarized_entities
                                        entity_graph
0  <graphml xmlns="http://graphml.graphdrawing.or...
❌ create_base_entity_graph
None
⠸ GraphRAG Indexer 
├── Loading Input (InputFileType.text) - 1 files loaded (1 filtered) ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00 0:00:00
├── create_base_text_units
├── create_base_extracted_entities
├── create_summarized_entities
└── create_base_entity_graph
❌ Errors occurred during the pipeline run, see logs for more details.

**Comment by AxithChoudhary** [comment, marked as answer]
go to logs and give what error you are getting . 
how to reach logs :
 output > reports > log
